src/extras/geometries/BoxGeometry.js

```diff
@@ -124,4 +124,21 @@ THREE.BoxGeometry = function ( width, height, depth, widthSegments, heightSegmen
 THREE.BoxGeometry.prototype = Object.create( THREE.Geometry.prototype );
 THREE.BoxGeometry.prototype.constructor = THREE.BoxGeometry;
 
+THREE.BoxGeometry.prototype.clone = function () {
+
+	var geometry = new THREE.BoxGeometry(
+		this.parameters.width,
+		this.parameters.height,
+		this.parameters.depth,
+		this.parameters.widthSegments,
+		this.parameters.heightSegments,
+		this.parameters.depthSegments
+	);
+
+	THREE.Geometry.prototype.clone.call( this, geometry );
```

Comment:
@mrdoob What happens when you re-comment on a previously-outdated diff -- as I am now also doing? It is collapsed and not visible... I wonder if the OP is notified somehow?...